plugins/module_utils/network/sonic/config/login_lockout/login_lockout.py

```diff
@@ -0,0 +1,272 @@
+        result['commands'] = commands
+
+        changed_login_lockout_facts = self.get_login_lockout_facts()
+
+        result['before'] = existing_login_lockout_facts
+
+        if result['changed']:
+            result['after'] = changed_login_lockout_facts
+        result['warnings'] = warnings
+
+        return result
+
+    def set_config(self, existing_login_lockout_facts):
+        """ Collect the configuration from the args passed to the module,
+            collect the current configuration (as a dict from facts)
+
+        :rtype: A list
+        :returns: the commands necessary to migrate the current configuration
+                  to the desired configuration
+        """
+        want = self._module.params['config']
+
+        have = existing_login_lockout_facts
+
+        resp = self.set_state(want, have)
+
+        return to_list(resp)
+
+    def set_state(self, want, have):
+        """ Select the appropriate function based on the state provided
+
+        :param want: the desired configuration as a dictionary
+        :param have: the current configuration as a dictionary
+        :rtype: A list
+        :returns: the commands necessary to migrate the current configuration
+                  to the desired configuration
+        """
+        state = self._module.params['state']
+        diff = get_diff(want, have)
+        if state == 'deleted':
+            commands, requests = self._state_deleted(want, have, diff)
+        elif state == 'merged':
+            commands, requests = self._state_merged(diff)
+        elif state == 'replaced':
+            commands, requests = self._state_replaced_overridden(want, have, diff)
+        elif state == 'overridden':
+            commands, requests = self._state_replaced_overridden(want, have, diff)
+        return commands, requests
+
+    def _state_replaced_overridden(self, want, have, diff):
+        """ The command generator when state is replaced or overridden
+
+        :rtype: A list
+        :returns: the commands necessary to migrate the current configuration
+                  to the desired configuration
+        """
+        commands = []
+        requests = []
+        delete = {}
+        state = self._module.params['state']
+
+        if (state == 'overridden'):
+            commands = diff
+            requests.extend(self.get_modify_specific_login_lockout_param_requests(commands))
+            commands = update_states(commands, 'overridden')
+        else:
+            for key in have:
+                if key not in diff:
+                    delete[key] = have[key]
+            if delete:
+                commands = delete
+                requests.extend(self.get_delete_specific_login_lockout_param_requests(commands))
+            if diff:
+                commands = diff
+                requests.extend(self.get_modify_specific_login_lockout_param_requests(commands))
+            commands = update_states(commands, 'replaced')
+
+        return commands, requests
```

Comment:
Since this module has a single instance of simple attributes, the result of replaced and overridden state will be same.
Please update the handling accordingly.